Imperium - Space Marines.cat

```diff
@@ -26794,6 +26949,21 @@ Crewed Artillery:After this unit is set up on the battlefield for the first time
         <characteristic name="Abilities" typeId="837d-5e63-aeb7-1410">-</characteristic>
       </characteristics>
     </profile>
+    <profile id="chief_librarian_profile" name="Chief Librarian" hidden="false" typeId="72c5eafc-75bf-4ed9-b425-78009f1efe82" typeName="Abilities">
```

Comment:
@script-driver why did you name IDs? It's not a thing. They're supposed to be random.

Main reason is that we never know what in the future the app author will do with these IDs. There have been migrations where IDs were generated for new entities based on old ones, appending some magical strings.

It's not safe. You should re-generate these IDs.

Reply:
I mainly did it for convenience, instead of having to copy-paste an ID I can just do it from memory. It saves time for me

Reply:
IDs are re-randomized.